### DISCUSSION TITLE
alist 在iPad min显示空白？

### DISCUSSION THREAD

**Discussion by gxjasvny** [AlistGo/alist#3966] [General]
alist在iPad min上Safai，Edge都显示空白？是需要哪里设置才能显示吗？

**Comment by Rektyfikowany** [comment]
访问`ping`接口(`http://主机地址:5244/ping`)应该显示`pong`，不显示请检查网络和防火墙，ping正常无法显示网页需提供浏览器日志以便排查

**Comment by gxjasvny** [comment]
是显示pong，但pong字外全是空白。好像alist v2.***时我的ipad是显示的，现在升到v3.14不显示了，我感觉v3.14网页输出的代码特意变动过的，好多别人alist.***.com我的ipad也打不开空白，但不是全部空白的，也有可以打开的，在我电脑是都可以打的。

**Comment by Rektyfikowany** [comment]
v2和v3的代码是不兼容。
最好参考[调试IOS Safari中的页面](https://blog.csdn.net/huahao1989/article/details/108020899)给出控制台的报错信息，或者留下你的ios版本看看有没有人能复现出你的问题

**Reply by gxjasvny** [reply]
我用的是ipad min2 IOS是10.0.2。已经试过了Safari 浏览器，选取“Safari 浏览器”>“偏好设置”，点按“高级”面板，清除历史网上数据等，不只是Safari，只要装在ipad上的浏览器 alist 内容都不显示。

**Comment by xhofe** [comment]
v3前端中用到了AsyncIterableIterator，所以你的浏览器至少需要支持es9的语法。

Safari版本大约需要大于12

**Comment by Sakana-1314** [comment]
我遇到了相同的问题, 请问你解决了吗?
我使用的是 IPad Air5,  safari 是最新版的, 我访问 alist 也是一片空白. 我的通过 Docker 安装 alist , 安卓和Windows都能正常使用

**Comment by gxjasvny** [comment]
ios>12
